src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -91,11 +93,21 @@ export default class DiscussionArea {
     this.#currentTimeMs = timeMs;
     DiscussionArea.#ELEMENT_TIMESTAMP_SPAN.innerText =
         timestampToString(timeMs);
-    const nearestComments = this.getNearestDiscussionComments(timeMs);
-    if (nearestComments.length == 0) {
+
+    // Remove any existing highlights.
+    for (const comment of this.#nearestComments) {
```

Comment:
Can you move this for loop into set highlighted?

Reply:
individual comments don't have access to nearestcomments